src/Mod/Draft/DraftEdit.py

```diff
@@ -932,10 +938,10 @@ def smoothBezPoint(self, point, style='Symmetric'):
                     return
                 if knot:
                     if style == 'Tangent':
-                        pts[changep] = self.obj.Proxy.modifytangentpole(\
+                        pts[changep] = obj.Proxy.modifytangentpole(\
```

Comment:
No need to use backslash inside parentheses, as the parentheses break the line correctly.
```py
pts[changep] = obj.Proxy.modifytangentpole(pts[knot], pts[keepp], pts[changep])
```

Or
```py
pts[changep] = obj.Proxy.modifytangentpole(pts[knot],
                                           pts[keepp], pts[changep])
```